src/services/evm/crawl_contract_evm.service.ts

```diff
@@ -152,11 +152,16 @@ export default class CrawlSmartContractEVMService extends BullableService {
 
     await knex.transaction(async (trx) => {
       if (evmContracts.length > 0) {
-        await EVMSmartContract.query()
-          .insert(evmContracts)
-          .onConflict('address')
-          .ignore()
-          .transacting(trx);
+        const newEvmContracts: EVMSmartContract[] = [];
+        evmContracts.forEach((evmContract, index) => {
+          const foundIndex = evmContracts
```

Comment:
đoạn này là sao nhỉ? là unique list contract address này à?
anh tưởng là so sánh với trong db
còn nếu unique cái list này thì em dùng uniqBy của lodash đi

Reply:
mấy cái uniq lodash nó toàn ra list string thôi ý anh, em cần list object cơ

Reply:
fixed